Source/Core/DolphinQt2/Settings/GeneralPane.cpp

```diff
@@ -38,6 +38,8 @@ void GeneralPane::CreateLayout()
   CreateAnalytics();
 #endif
   CreateAdvanced();
+
+  m_main_layout->setMargin(0);
```

Comment:
This is deprecated, and `setContentsMargins(0, 0, 0, 0)` should be used instead: http://doc.qt.io/qt-4.8/qlayout-obsolete.html#margin-prop